bin/lambda-entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+if [ $# -ne 1 ]; then
+  echo "entrypoint requires the handler name to be the first argument" 1>&2
+  exit 142
+fi
+export _HANDLER="$1"
+
+# Set API_ACCESS_KEY if api_access_key exists
+if [ -f "${LAMBDA_TASK_ROOT}/.api_access_key" ]; then
+  export API_ACCESS_KEY=$(cat ${LAMBDA_TASK_ROOT}/.api_access_key)
+  rm -f ${LAMBDA_TASK_ROOT}/.api_access_key
+fi
```

Comment:
`lambda-entrypoint.sh` is called as the ENTRYPOINT, which is executed when the image is started. 
Added this section, where it reads the `.api_access_key` file into the `API_ACCESS_KEY` environment variable. Then removes the file